zrml/prediction-markets/src/tests.rs

```diff
@@ -243,7 +251,7 @@ fn admin_destroy_market_correctly_slashes_permissionless_market_reported() {
             0..end,
             ScoringRule::CPMM,
         );
-        run_to_block(end);
+        run_to_block(end + BLOCKS_PER_DAY + 1);
```

Comment:
I would love to see the value according to the market here instead of using the raw values here. Please search for `BLOCKS_PER_DAY`or `BLOCKS_PER_DAY_U32` to correct all of these.